test/wallet-http-test.js

```diff
@@ -85,6 +92,45 @@ for (const witnessOpt of witnessOptions) {
       addr = Address.fromString(str, node.network);
     });
 
+    it('should change passphrase', async () => {
+      await wallet.setPassphrase('initial');
+      await wclient.lock('test');
+
+      // Incorrect Passphrase should not work
+      try {
+        await wallet.unlock('badpass');
+        throw new Error('Unlocked with INCORRECT passphrase!');
+      } catch(e) {
+        assert.strictEqual(e.message, 'Could not decrypt.', e.message);
+      }
```

Comment:
This construction can be replaced using `assert.rejects()` grep the other tests to find an example.

Reply:
interesting!